.cargo/config.toml

```diff
@@ -14,3 +14,11 @@ rustflags = [
     # in line with Linux (whereas default for Windows is 1MB)
     "-Clink-arg=/STACK:10000000",
 ]
+
+
+[patch."https://github.com/bitfinity-network/bitfinity-evm-sdk"]
```

Comment:
to be removed.
If there are required changes in the bitfinity-sdk-evm create a PR

Reply:
Yeah. This is how I use not published modifications from other repos. There are no new changes required, just yours.

Usually, .cargo  folder is ignored as it's supposed to be used for local configs... This is why I forgot not to add it to the commit.

I'll remove it after sdk changes are merged.